frontend-react/src/components/DataDashboard/ReportDetails/ReportDetailsTable.tsx

```diff
@@ -0,0 +1,93 @@
+import React from "react";
+
+import { useReportsFacilities } from "../../../hooks/network/DataDashboard/DataDashboardHooks";
+import Table, { TableConfig } from "../../../components/Table/Table";
+import { EventName, trackAppInsightEvent } from "../../../utils/Analytics";
+import TableFilters from "../../Table/TableFilters";
+import useFilterManager, {
+    FilterManagerDefaults,
+} from "../../../hooks/filters/UseFilterManager";
+import { FeatureName } from "../../../AppRouter";
+
+import styles from "./ReportDetailsTable.module.scss";
+
+const filterManagerDefaults: FilterManagerDefaults = {
+    sortDefaults: {
+        column: "collectionDate",
+        order: "DESC",
+    },
+};
+
+interface ReportDetailsTableProps {
+    reportId: string;
+}
+
+function ReportDetailsTable(props: ReportDetailsTableProps) {
+    const { reportId }: ReportDetailsTableProps = props;
+    const { data } = useReportsFacilities(reportId);
+    const featureEvent = `${FeatureName.REPORT_DETAILS} | ${EventName.TABLE_FILTER}`;
+
+    const filterManager = useFilterManager(filterManagerDefaults);
+
+    const tableConfig: TableConfig = {
+        columns: [
+            {
+                dataAttr: "provider",
+                columnHeader: "Ordering provider",
+                feature: {
+                    link: true,
+                    linkBasePath: "/data-dashboard/facilities-providers/",
```

Comment:
worth turning `/data-dashboard/facilities-providers/` into a variable in the `site.json` file? https://github.com/CDCgov/prime-reportstream/blob/master/frontend-react/src/content/site.json

Reply:
@etanb this will be a link to the FacilitiesProviders.tsx once that is ready so I don't think it needs to be a variable.